### PR TITLE
Update changeDate when updating metadata

### PR DESCRIPTION
**Pull request by Vampouille**

### CODE REVIEW

File: core/src/main/java/org/fao/geonet/kernel/DataManager.java [hunk 1]
```diff
@@ -2066,9 +2066,13 @@ public Metadata updateMetadata(final ServiceContext context, final String metada
         Integer intId = Integer.valueOf(metadataId);
         if (ufo) {
             String parentUuid = null;
-            metadataXml = updateFixedInfo(schema, Optional.of(intId), null, metadataXml, parentUuid, (updateDateStamp ? UpdateDatestamp
-
-                    .YES : UpdateDatestamp.NO), context);
+            metadataXml = updateFixedInfo(schema,
```
Comment (fgravin) [inline core/src/main/java/org/fao/geonet/kernel/DataManager.java]: What is the change here ?

Comment (Vampouille) [inline core/src/main/java/org/fao/geonet/kernel/DataManager.java]: This just improve readability

Comment (fgravin) [inline core/src/main/java/org/fao/geonet/kernel/DataManager.java]: Yeah but it will make the merge from geonetwork even harder.

Comment (fgravin) [inline core/src/main/java/org/fao/geonet/kernel/DataManager.java]: Please revert the identation optimisation, to stay closer from upstream.
thanks

Comment (Vampouille) [inline core/src/main/java/org/fao/geonet/kernel/DataManager.java]: I just remove identation optimisation.

**Pull request merged**